### PR TITLE
Solved #14 - loading center issue

### PR DESCRIPTION
**Pull request by ankitgauri26**
Corrected the CSS for loading.

### CODE REVIEW

**Comment by ankitgauri26** [comment]
@MaathavanJkr - Solved the issue. Kindly review

**Comment by MaathavanJkr** [comment]
Nice. But the Loading can be more centered in Mobile Versions. in mobile.html rank.html study.html. 

see
வரைபடவி - Varaipadavi - Google Chrome 10_22_2020 2_18_53 AM (2)

**Comment by ankitgauri26** [comment]
@MaathavanJkr - Loading seems to be centered in the Screenshot shared by you. Do you want to move loading up in "mobile.html rank.html study.html.".
I have changed the CSS which will effect in all the files and hence loading is coming as centered.. Previously when you zoom out, loading text is not aligned with loading icon. I have corrected that.
If still you want me to correct anything, request you to please elaborate more as loading seems centered for me in the screen shot above. What exactly needs to be done according to your screen shot

**Comment by Matrix278** [comment]
I think he wanted to have some space on mobile, as now it looks not kinda in center of mobile phone, like here https://github.com/kanimaitech/Varaipadavi/pull/18#issuecomment-713754358, I think he wants that padding space with that icon

**Comment by ankitgauri26** [comment]
@Matrix278 @MaathavanJkr - What I understand is loading icon and loading text should be in the center for all elements i.e. in the above screen it is in center from 3 sides (left, right and top) but not from bottom as it is very near to show button. Please let me know if my understanding is correct so that I can work on this?

**Comment by MaathavanJkr** [comment]
See the dialog box. The loading icon is not centered vertically. The above
gap is greater than the below gap.

Maathavan

**Comment by ankitgauri26** [comment]
@MaathavanJkr - Thank you for the clarification. I'll work on it. Thanks

**Comment by MaathavanJkr** [comment]
and if you can reduce the gap between the icon and loading text. Thank you

**Comment by ankitgauri26** [comment]
Corrected the CSS now. Kindly review
image

**Comment by ankitgauri26** [comment]
@MaathavanJkr - Kindly review the commits in the PR. Done 2 commit. 1 previous and 1 today.

**Comment by ankitgauri26** [comment]
@MaathavanJkr - Also reduced the gap between the icon and loading text. Thank you

File: css/common.css [hunk 1]
```diff
@@ -422,4 +423,18 @@ td.rank {
 	-webkit-transform: rotate(45deg);
 	-ms-transform: rotate(45deg);
 	transform: rotate(45deg);
+}
+
+
+#quiz-loader,
```
Comment (MaathavanJkr) [inline css/common.css:429]: Instead of this we can just add text-center class to these elements?

File: rank.html [hunk 2]
```diff
@@ -49,7 +49,7 @@
             <div></div>
             <div></div>
         </div>
-        <div class="text-center">Loading</div>
+        <div class="loading-text text-center">Loading</div>
```
Comment (MaathavanJkr) [inline rank.html:52]: then we wont need this text-center class here ig

**Review by MaathavanJkr — CHANGES_REQUESTED** [review]
Awesome work. :) Consider this too ;)

File: css/common.css (hunk 1 above)
Comment (ankitgauri26) [reply css/common.css]: Added "text-center" class to these elements. Thank you for the review

File: rank.html (hunk 2 above)
Comment (ankitgauri26) [reply rank.html]: Removed text-center from loading as we have applied text-center in parent element. Thank you for the review

**Comment by ankitgauri26** [comment]
@MaathavanJkr - Done the changes as per review. Thank you again for the suggestions.

**Comment by MaathavanJkr** [comment]
@ankitgauri26 Great Work. Thank you. :)

**Pull request merged**